query/aggregator.go

```diff
@@ -146,7 +160,14 @@ func applyMul(a, b, c *types.Val) error {
 	vBase := getValType(a)
 	switch vBase {
 	case INT:
-		c.Value = a.Value.(int64) * b.Value.(int64)
+		aVal, bVal := a.Value.(int64), b.Value.(int64)
+		c.Value = aVal * bVal
+
+		if aVal == 0 || bVal == 0 || aVal == 1 || bVal == 1 {
```

Comment:
Remove checks for 1 and minInt64 as they are handled in `c.Value.(int64)/bVal != aVal` case.